run.py

```diff
@@ -2,16 +2,18 @@
 """Main runnable file for the codebase"""
 
 import logging
-
+import sys
 from state_machine.flight_manager import FlightManager
 
-
 if __name__ == "__main__":
     # Run multiprocessing function
     try:
+        simflag: bool = False
```

Comment:
Variable name should adhere to snake_case:
```suggestion
        sim_flag: bool = False
```